decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -158,6 +158,14 @@ def remove_default_error_pages
       def decidim_initializer
         copy_file "initializer.rb", "config/initializers/decidim.rb"
 
+        gsub_file "config/environments/production.rb",
+                  /config.log_level = :debug/,
+                  "config.log_level = %w(debug info warn error fatal).include?(ENV['RAILS_LOG_LEVEL']) ? ENV['RAILS_LOG_LEVEL'] : :debug"
+
+        gsub_file "config/environments/production.rb",
+                  %r{# config.action_controller.asset_host = 'http://assets.example.com'},
```

Comment:
```suggestion
                  %r{# config.action_controller.asset_host = 'http://assets.example.org'},
```

Reply:
> @microstudi I've built the docs site to see how this will look and to catch if we've made any formatting issue. I also replaced the statics references to hotlink from the prod website. I didn't find any issues. If you want, you can give it a look too:
> 
> [environment_variables.zip](https://github.com/decidim/decidim/files/7969662/environment_variables.zip)

It looks pretty good. I wonder if it would be possible to expand the description column to give the explanation a little bit more horizontal room